bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -597,6 +578,8 @@ yerr_low, yerr_high, fraction_mc = \
 red_efficiency = (fraction_mc) - (yerr_low) * scipy.stats.norm.isf(0.1)
 
 # Calculate and save to disk 50% and 90% exclusion distances
+# excl_dist dictionary contains 50% and 90% exclusion distances 
+excl_dist = {}
```

Comment:
It looks like you can use this dictionary only, and remove `excl_dist_dict`.  Details below.